lib/elixir/lib/module.ex

```diff
@@ -1190,8 +1190,12 @@ defmodule Module do
     end
   end
 
-  defp preprocess_attribute(:on_load, atom) when is_atom(atom) do
-    {atom, 0}
+  defp preprocess_attribute(:on_load, value) do
```

Comment:
Oh missed that, that's right. We don't document it in the docs for Module though, should we?